TestHost/Program.cs

```diff
@@ -135,12 +128,16 @@ private static void DiscoverUnitTests(IEnumerable<ITestDiscoverer> tds, string s
                 timeFilter,
                 (string assemblyPath) =>
                 {
-                    var asmPath = FilePath.NewFilePath(assemblyPath);
                     var disc = new XUnitTestDiscoverer();
                     disc.TestDiscovered.AddHandler(
                         new FSharpHandler<TestCase>(
                             (o, ea) =>
                             {
+                                if (ignoredTests.Any(x => ea.FullyQualifiedName.StartsWith(x)))

```

Comment:
Isnt this callback called for all kinds of test explorers? We dont want to put anything in specific test adapters.